lib/lotus/generators/application/container.rb

```diff
@@ -21,12 +22,13 @@ def initialize(command)
         def start
 
           opts      = {
-            app_name:            app_name,
-            lotus_head:          @lotus_head,
-            test:                @test,
-            database:            @database,
-            database_config:     database_config,
-            lotus_model_version: @lotus_model_version
+            app_name:               app_name,
+            lotus_head:             @lotus_head,
+            test:                   @test,
+            database:               @database,
+            database_config:        database_config,
+            lotus_model_version:    @lotus_model_version,
+            lotus_helpers_version:  @lotus_helpers_version
```

Comment:
can we remove 1 space in between gem 'lotus-helpers', and '<%= config[:lotus_helpers_version] %>'?